### PR TITLE
docs(linux): Power_Management: LPM documentation cleanups

### PR DESCRIPTION
**Pull request by kwillis01**
Move a detailed latency constraint from device constraint section to the latency constraint section. Fix the MCU GPIO wakeup pin which was documented incorrectly. Cleanup capitalization and whitespace errors.

### CODE REVIEW

**Review by vishalmti — APPROVED** [review]

File: source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst [hunk 1]
```diff
@@ -222,7 +222,7 @@ Now, under the switch node we add the following:
     .. code-block:: console
 
         root@<machine>:~# cat /proc/interrupts | grep "MCUGPIO"
-        262:          0          0          0          0      GPIO  16 Edge    -davinci_gpio  MCUGPIO
+        273:          0          0          0          0      GPIO  4 Edge    -davinci_gpio  MCUGPIO
```
Comment (StaticRocket) [inline source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:225]: Is the `4` here some specific pin? If it is, then this section is not machine agnostic and should actually say what machine it is in the code-block.

File: source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst [hunk 2]
```diff
@@ -242,8 +242,8 @@ MCU GPIO wakeup can only be tested when
 overlay is loaded. Please refer to :ref:`How to enable DT overlays<howto_dt_overlays>` for more details.
 
 Once the system has entered Deep Sleep or MCU Only mode as shown in the
-:ref:`LPM section<lpm_modes>`, wakeup from MCU GPIO0_16 can be triggered
-by grounding Pin 11 on J8 MCU Header.
+:ref:`LPM section<lpm_modes>`, wakeup from MCU_SPI0_D1 can be triggered
+by grounding Pin 4 on J8 MCU Header.
```
Comment (StaticRocket) [inline source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:246]: Isn't this device specific? This file is included by AM62 AM62A and AM62P configs. It should `ifconfig`ed out for each platform if you want to mention the specific pin.

**Review by StaticRocket — CHANGES_REQUESTED** [review]
Most of this is useful. Just a nitpick about pins and device specific sections.

File: source/linux/Foundational_Components/Power_Management/pm_dfs.rst [hunk 3]
```diff
@@ -332,6 +332,7 @@ table.
                                 };
 
         To disable any of the above OPP's one can simply add
+
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Power_Management/pm_dfs.rst:335]: The indentation of code-blocks directive doesn’t follow - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst [hunk 4]
```diff
@@ -200,6 +195,11 @@ Default constraint is "no constraint", but it can be changed as shown in the exa
 Setting a resume latency constraint impacts the deepest low power mode system can enter.
 The various modes and their latencies are documented here: https://downloads.ti.com/tisci/esd/latest/2_tisci_msgs/pm/lpm.html#tisci-msg-lpm-set-latency-constraint
 
+If a device wants to put a constraint to be not powered-off, it can use the Linux
+QoS framework and set the DEV_PM_QOS_RESUME_LATENCY equal to 0.
+An example is shown in the following RemoteProc driver:
+https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/drivers/remoteproc/ti_k3_r5_remoteproc.c?h=10.00.07#n535
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst:201]: Links should be w.r.t. `10.01.10` tag

**Review by cshilwant — DISMISSED** [review]
Check if all below references are still valid, if they are use `10.01.10` tag in URLs

`linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:182:`k3-am62x-sk-lpm-wkup-sources.dtso <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62x-sk-lpm-wkup-sources.dtso?h=10.00.07>`

`linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:241:`k3-am62x-sk-lpm-wkup-sources.dtso <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62x-sk-lpm-wkup-sources.dtso?h=10.00.07>`

`linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:284:main_uart0 node in `k3-am62x-sk-common.dtsi <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi?h=10.00.07>`

`linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:347:gpio_key node in `k3-am62x-sk-lpm-wkup-sources.dtso <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62x-sk-lpm-wkup-sources.dtso?h=10.00.07#n21>`

`linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:350:`k3-am62x-sk-lpm-wkup-sources.dtso <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62x-sk-lpm-wkup-sources.dtso?h=10.00.07>`

`linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:374:`k3-am62-wakeup.dtsi <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62-wakeup.dtsi?h=10.00.07#n36>`

`linux/Foundational_Components/Power_Management/pm_sw_arch.rst:154:#. TISCI PM Domain driver: https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/drivers/pmdomain/ti/ti_sci_pm_domains.c?h=10.00.07`

`linux/Foundational_Components/Power_Management/pm_sw_arch.rst:155:#. TISCI driver: https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/drivers/pmdomain/ti/ti_sci_pm_domains.c?h=10.00.07`

`linux/Foundational_Components/Power_Management/pm_sw_arch.rst:180:https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/drivers/remoteproc/ti_k3_r5_remoteproc.c?h=10.00.07#n535`

`linux/Foundational_Components/Power_Management/pm_low_power_modes.rst:241:when `k3-am62x-sk-lpm-wkup-sources.dtso <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62x-sk-lpm-wkup-sources.dtso?h=10.00.07>`

`linux/Foundational_Components/Kernel/Kernel_Drivers/E5010_JPEG_Encoder.rst:16:  <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/drivers/media/platform/imagination/e5010/e5010-jpeg-enc.c?h=10.00.07>`

`linux/Foundational_Components/U-Boot/UG-Splash-Screen.rst:219:  * `Splash screen config fragment for AM62x and AM62P  <https://git.ti.com/cgit/ti-u-boot/ti-u-boot/tree/configs/am62x_a53_splashscreen.config?h=10.00.07>`

`linux/Foundational_Components/U-Boot/UG-Splash-Screen.rst:226:  * `Splash screen config fragment for AM62x and AM62P  <https://git.ti.com/cgit/ti-u-boot/ti-u-boot/tree/configs/am62x_a53_splashscreen.config?h=10.00.07>`

File: source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst [hunk 5]
```diff
@@ -200,6 +195,11 @@ Default constraint is "no constraint", but it can be changed as shown in the exa
 Setting a resume latency constraint impacts the deepest low power mode system can enter.
 The various modes and their latencies are documented here: https://downloads.ti.com/tisci/esd/latest/2_tisci_msgs/pm/lpm.html#tisci-msg-lpm-set-latency-constraint
 
+If a device wants to put a constraint to be not powered-off, it can use the Linux
```
Comment (AashvijShenai) [inline source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst:198]: to be not powered-off -> to not be powered-off

File: source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst (hunk 2 above)
Comment (kwillis01) [reply source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:246]: this is not device specific to AM62 AM62A and AM62P. They all use the same overlay to create this wakeup source, so pin 4 is the correct pin for each of them

File: source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst (hunk 4 above)
Comment (kwillis01) [reply source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst]: fixed it to be 10.01.10 tag

File: source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst (hunk 5 above)
Comment (kwillis01) [reply source/linux/Foundational_Components/Power_Management/pm_sw_arch.rst]: fixed the wording to the suggested

**Comment by cshilwant** [comment]
@DhruvaG2000 @akashdeep-ti for reviews

**Review by StaticRocket — DISMISSED** [review]

File: source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst [hunk 6]
```diff
@@ -301,26 +301,26 @@ Setup the interrupt parent and interrupt as MCU_GPIO0,
 
 .. code-block:: dts
 
-   interrupt-parent = <&mcu_gpio0>;
-   interrupts = <16 IRQ_TYPE_EDGE_RISING>;
+    interrupt-parent = <&mcu_gpio0>;
```
Comment (akashdeep-ti) [inline source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst:304]: https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

**Review by StaticRocket — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Pull request merged**